server/src/weaverbird/backends/pypika_translator/translators/googlebigquery.py

```diff
@@ -54,23 +63,24 @@ class GoogleBigQueryQueryBuilder(QueryBuilder):
     QUERY_CLS = GoogleBigQueryQuery
 
     def __init__(self, **kwargs: Any) -> None:
-        super().__init__(dialect=SQLDialect.GOOGLEBIGQUERY, **kwargs)
+        super().__init__(
+            dialect=SQLDialect.GOOGLEBIGQUERY, wrapper_cls=GoogleBigQueryValueWrapper, **kwargs
+        )
+
+    @staticmethod
+    def wrap_constant(val, wrapper_cls=GoogleBigQueryValueWrapper):
+        return super().wrap_constant(val, wrapper_cls)
+
+    @staticmethod
+    def wrap_json(val, wrapper_cls=GoogleBigQueryValueWrapper):
+        return super().wrap_json(val, wrapper_cls)
```

Comment:
a small UT to represent these new changes?

Reply:
It's still a WIP yes don't worry!